<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1232</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CircuitSim</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">tabWidget::close-button{
width: 13px;
height: 13px;
background-color: rgb(255, 0, 255);
}</string>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::TabShape::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>16777</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Ferramentas</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="10" column="0">
          <widget class="QToolButton" name="btnAdicionarNOT">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QToolButton" name="btnSelecao">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QToolButton" name="btnAdicionarAND">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QToolButton" name="btnInteracao">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QToolButton" name="btnAdicionarInput">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QToolButton" name="btnAdicionarOutput">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QToolButton" name="btnConexao">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QToolButton" name="btnLixeira">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QToolButton" name="btnAdicionarOR">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1232</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSalvar_como"/>
    <addaction name="actionClose_tab"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Sobre</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>Novo</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Ajuda</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionSalvar_como">
   <property name="text">
    <string>Salvar como</string>
   </property>
  </action>
  <action name="actionClose_tab">
   <property name="text">
    <string>Fechar aba</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
